instrumentation/jaxrs-client/jaxrs-client-2.0/jaxrs-client-2.0-jersey-2.0/src/main/java/io/opentelemetry/javaagent/instrumentation/jaxrsclient/v2_0/JerseyClientInstrumentationModule.java

```diff
@@ -104,56 +101,4 @@ public static void handleError(
     }
   }
 
-  public static class WrappedFuture<T> implements Future<T> {
```

Comment:
Test was failing since groovy eagerly calls `getEnclosingClass()` on almost everything! But technically this was a bug in our agent since there's nothing stopping (other than mostly having no use case for it) a user from calling `getEnclsoingClass()` too. As a general pattern we should avoid helper classes that are inner to the instrumentation.